boards/Dell/OptiPlex_7010_9010/results.csv

```diff
@@ -0,0 +1,197 @@
+USB devices detection after cold boot (Ubuntu),SUD0001.001,PASS
+USB devices detection after warm boot (Ubuntu),SUD0002.001,PASS
+USB devices detection after reboot (Ubuntu),SUD0003.001,PASS
+USB devices detection after suspension (Ubuntu),SUD0004.001,PASS
+USB devices detection after suspension (Ubuntu) (S3),SUD0004.003,PASS
+NVMe detection after cold boot (Ubuntu),SNV0001.001,PASS
+NVMe detection after warm boot (Ubuntu),SNV0002.001,PASS
+NVMe detection after reboot (Ubuntu),SNV0003.001,PASS
+NVMe detection after suspension (Ubuntu),SNV0004.001,PASS
+NET controller after coldboot (Ubuntu),NET001.001,PASS
+NET controller after warmboot (Ubuntu),NET002.001,PASS
+NET controller after reboot (Ubuntu),NET003.001,PASS
+NET controller after suspend (Ubuntu),NET004.001,PASS
+NET controller after suspend (Ubuntu) (S3),NET004.003,PASS
+Detect TPM after platform reboot (Ubuntu),TPD003.001,PASS
+Detect TPM after platform suspend (Ubuntu),TPD004.001,PASS
+Netboot is available,NBT001.001,PASS
+OS Selection & Utilities is available,NBT002.001,PASS
+iPXE boot is available,NBT003.001,PASS
+iPXE shell is available,NBT004.001,PASS
+iPXE shell works correctly,NBT005.001,PASS
+USB detection after coldboot,UDT001.001,PASS
+USB detection after warmboot,UDT001.002,PASS
+USB detection after system reboot,UDT001.003,PASS
+USB detect and boot after coldboot,UBT001.001,PASS
+USB detect and boot after warmboot,UBT002.001,PASS
+USB detect and boot after system reboot,UBT003.001,PASS
+Verify if no reboot occurs in the firmware,STB001.001,PASS
+Verify if no reboot occurs in the OS (Ubuntu),STB001.002,PASS
+Verify if no unexpected boot errors appear in Linux logs,STB002.001,PASS
+CPU clock speed displayed in setup menu,SET001.001,PASS
+RAM speed displayed in setup menu,SET002.001,PASS
+RAM size displayed in setup menu,SET003.001,PASS
+Expected CPU clock speed displayed in setup menu,SET004.001,PASS
+Expected RAM speed displayed in setup menu,SET005.001,PASS
+Expected RAM size displayed in setup menu,SET006.001,PASS
+Boot Ubuntu LTS from disk after cold-boot,BUB001.001,PASS
+Boot Ubuntu LTS from disk after warm-boot,BUB002.001,PASS
+Boot Ubuntu LTS from disk after reboot,BUB003.001,PASS
+Boot Windows 11 from disk after cold-boot,BWI001.001,PASS
+Boot Windows 11 from disk after warm-boot,BWI002.001,PASS
+Boot Windows 11 from disk after reboot,BWI003.001,PASS
+CPU works (Ubuntu),CPU001.001,PASS
+CPU works (Windows),CPU001.002,PASS
+CPU cache enabled (Ubuntu),CPU002.001,PASS
+Multiple CPU support (Ubuntu),CPU003.001,PASS
+Multiple-core support (Ubuntu),CPU004.001,PASS
+Change Auto Boot Time-out and check after reboot,BMM001.001,PASS
+F9 resets Auto Boot Time-out to default value,BMM002.001,PASS
+Check Auto Boot Time-out option not accept non-numeric values,BMM003.001,PASS
+EC power button watchdog,ECR033.001,PASS
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
+,,
```

Comment:
Remove unnecessary empty cells

Reply:
That's an error in the `update_results.py` script, there is a PR fixing it here https://github.com/Dasharo/osfv-results/pull/9

https://github.com/Dasharo/osfv-results/compare/bafe39bbc5a79ee19b50f689fe69fea5002f891b..c79b8fc895dd83df865ee79ce902c7f9e7921281#diff-3bb95ffd5bea876e4f909e367c3747236e1d3b97ecca8d2ebbc64e90830f9551L162-L197